website/blog/2022-05-19-GAAD-2022-update.md

```diff
@@ -0,0 +1,48 @@
+---
+title: React Native Accessibility - GAAD 2022 Update
+author: Alex Tait
+authorTitle: React Native Accessibility Community Manager
+authorURL: 'https://twitter.com/AT_Fresh_Dev'
+authorImageURL: 'https://github.com/alextait1.png'
+authorTwitter: AT_Fresh_Dev
+tags: [announcement]
+date: 2022-05-19
+---
+
+May 19th, 2022 marks the 11th annual celebration of Global Accessibility Awareness Day and we wanted to update everyone on the accessibility progress we’ve made on the React Native Framework. Meta (formerly Facebook) was the [first organization to take the GAAD pledge in 2020](/blog/2021/03/08/GAAD-React-Native-Accessibility), committing to making the React Native framework accessible.
+
+> _“We hope this pledge makes it easier for developers using React Native to create fully accessible mobile apps and inspires other organizations to make similar commitments to a more accessible future.”_
+>
+> — [Mike Shebanek, head of accessibility Meta, 2020](https://gaad.foundation/gaadpledge/)
```

Comment:
Also, if there is any admonition style which fits here, let's use it instead of the `blockquote`:
* https://docusaurus.io/docs/next/markdown-features/admonitions

Reply:
I was actually surprised to see the quote rendered as an admonition here. Don't we have something more suitable for a quote (like a horizontal bar on the side + italic only)?

Reply:
This is the legacy, from the time when there were no admonitions in Docusaurus. In those days we have used `blockquote` to create a distinguish "note" block. 

If we migrate the old notes to admonitions, we should be able to restyle `blockquote` back to it intended design, and usage purpose, but it is out of scope of this PR for sure. 😅